apps/form-management-app/src/app/components/autoComplete/json.ts

```diff
@@ -0,0 +1,184 @@
+import type { CancellationToken, editor, IRange, languages, Position } from 'monaco-editor';
+import type { EditorSuggestion } from './autoComplete';
+
+export abstract class JsonPropertyValueCompletionItemProvider implements languages.CompletionItemProvider {
```

Comment:
files under autoComplete are not UI components, if these are shared things can you move tehse to under utils? or put them under libs folder within a proper subfolder